src/components/molecules/GeoJsonForm.vue

```diff
@@ -45,6 +45,7 @@ export default defineComponent({
       } else {
         geoShape.value = jsonForm;
       }
+      geoInputShape.value = geoShape.value;
```

Comment:
Why do we need `geoShape` and `geoInputShape` as two separate refs? Could we name them more accurately to explain the distinction?

Reply:
We need it to be separate so we don't redraw the shape that a user drew on the map. Perhaps 'uploadedShape' is better?

Reply:
Oh, well then we should probably not assign to `geoShape` at all when we use the copy/paste or file upload inputs. Can we rename `geoShape` to `drawnShape` and `geoInputShape` to `specifiedShape` (Just because we haven't uploaded it yet but we will)?

Then these lines just become 
```
specifiedShape.value =  jsonForm;
```